Vagrantfile

```diff
@@ -115,65 +83,29 @@ else
 end
 
 if ENV['K8S'] then
-    $k8stag = ENV['K8STAG'] || "-k8s"
+    $vm_base_name = "k8s"
+else
+    $vm_base_name = "runtime"
 end
 
-# We need this workaround since kube-proxy is not aware of multiple network
-# interfaces. If we send a packet to a service IP that packet is sent
-# to the default route, because the service IP is unknown by the linux routing
-# table, with the source IP of the interface in the default routing table, even
-# though the service IP should be routed to a different interface.
-# This particular workaround is only needed for cilium, running on a pod on host
-# network namespace, to reach out kube-api-server.
-$kube_proxy_workaround = <<SCRIPT
-sudo iptables -t nat -A POSTROUTING -o enp0s8 ! -s 192.168.34.12 -j MASQUERADE
-SCRIPT
-
 Vagrant.configure(2) do |config|
     config.vm.provision "bootstrap", type: "shell", inline: $bootstrap
     config.vm.provision "build", type: "shell", run: "always", privileged: false, inline: $build
     config.vm.provision "install", type: "shell", run: "always", privileged: false, inline: $install
 
-    config.vm.provider :libvirt do |libvirt|
-        config.vm.box = "cilium/ubuntu-16.10"
-        config.vm.box_version = "2.0"
-        libvirt.memory = ENV['VM_MEMORY'].to_i
-        libvirt.cpus = ENV['VM_CPUS'].to_i
-        config.vm.synced_folder ".", "/home/vagrant/go/src/github.com/cilium/cilium", disabled: false
-    end
-
     config.vm.provider "virtualbox" do |vb|
         # Do not inherit DNS server from host, use proxy
         vb.customize ["modifyvm", :id, "--natdnshostresolver1", "on"]
         vb.customize ["modifyvm", :id, "--natdnsproxy1", "on"]
-
-        config.vm.box = "cilium/ubuntu-16.10"
-	config.vm.box_version = "2.7"
+        config.vm.box = "eloycoto/ubuntu_test"
```

Comment:
The images we provide should not be in a private user's account, only under `cilium/`.

Reply:
Discussed offline, this is due to issues with VagrantCloud uploading. We will use this as a stopgap until we get better storage resources for the official image.